src/dataService/tagDataService.ts

```diff
@@ -51,11 +51,12 @@ export class TagDataService {
       .select(
         'tag as name',
         'tag',
+        this.db.raw('MAX(id) as _cursor'),
```

Comment:
if we need a sortId to be mapped with the tag's cursor, would it make sense to get the first tag's cursor, so we don't parse the entire tag list of the user?

_(i might be missing some context on why we need to do max)_ may be, we can document why we chose max as comment

Reply:
using max here gets the tags in order they were most recently used. so if I have tagging behavior like this:

(ascending order from past) 

tag item A as 'recipe'
tag item B as 'work'
tag item C as 'recipe'

A.id < B.id < C.id

but in aggregate, 'recipe'.id > 'work'.id, so tags in most recently used order would show up appropriately


what you're describing is a limitation in the modeling... I agree here that it would be best if we didn't have to group all the same-named tags